crates/telio-nurse/src/nurse.rs

```diff
@@ -163,6 +152,22 @@ impl State {
         }
     }
 
+    /// Inform Nurse that Derp service started.
+    ///
+    /// # Arguments
+    ///
+    /// * `derp` - The new Derp service pointer to use.
+    pub async fn configure_meshnet(&self, meshnet_entities: Option<MeshnetEntities>) {
+        let _ = task_exec!(&self.heartbeat, async move |state| {
+            state.configure_meshnet(meshnet_entities).await;
+
+            telio_log_debug!("Updated meshnet config");
```

Comment:
Nit: it would be so nice to log whether we are enabling meshnet analytics or disabling here :thinking: